compiler/qsc_passes/src/lib.rs

```diff
@@ -154,3 +159,16 @@ pub fn run_core_passes(core: &mut CompileUnit) -> Vec<Error> {
         .chain(base_prof_errors.into_iter().map(Error::BaseProfCk))
         .collect()
 }
+
+pub fn run_fir_passes(
+    package: &fir::Package,
```

Comment:
just to confirm, the change to refactor the FIR lowering out of qsc_eval will be part of a later PR, right? At that point, we could consider pulling this up into the `PassContext` as the path for integration into the existing scenarios, but that's out of scope for now, making this effectively a test only API at the moment.